osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -145,6 +146,9 @@ public enum GlobalAction
         [Description("Decrease scroll speed")]
         DecreaseScrollSpeed,
 
+        [Description("Toggle in-game interface")]
+        ToggleInGameInterface,
```

Comment:
This needs to be at the very bottom of the enum, otherwise it offsets all other values and breaks existing keybinds. It's a known limitation.

Reply:
makes sense, sorry ill fix